src/imars3d/backend/corrections/denoise.py

```diff
@@ -154,11 +153,17 @@ def denoise_by_bilateral(
             # copy the data to the shared memory
             np.copyto(shm_arrays, arrays)
             # mp
+            kwargs = {
+                "max_workers": max_workers,
+                "desc": "denoise_by_bilateral",
+            }
+            if tqdm_class:
+                kwargs["tqdm_class"] = tqdm_class
```

Comment:
Test are not covering this line, which leads to the large decrease as it happens in multiple files.
Is it intended to skip testing for this type of statement?

Reply:
I'm not sure what we want to do on this one. I think it is an acceptable risk since the Tqdm widget is tested *and* we have repeats in the tilt correction unit tests.

Reply:
okay, I will resolve the other related conversations.